cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs

```diff
@@ -1527,6 +1528,14 @@ instance
     -- TODO: fix
     Conway.ScriptIntegrityHashMismatch _ _ -> undefined
 
+instance ToObject (Praos.PraosTiebreakerView crypto) where
+  toObject v (Praos.PraosTiebreakerView sl issuer issueNo vrf) =
+    mconcat [ "slotNo" .= toObject v sl
+            , "issuer" .= textShow issuer
+            , "issueNo" .= textShow issueNo
+            , "vrf" .= textShow vrf
+            ]
```

Comment:
This can be made a bit nicer (not using verbose `Show` instances) by adapting what is present on `main` for `PraosChainSElectView`:
https://github.com/IntersectMBO/cardano-node/blob/0f6b522cf720a298146875135cf91aa6043dbd74/cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs#L1330-L1346
(Also, I think it makes sense to use record field names to avoid future unexpected changes)